workers/ohsome_quality_analyst/utils/models.py

```diff
@@ -0,0 +1,66 @@
+"""
```

Comment:
Is this the right path/module for `models.py` to exist? Is the name chosen well?

Reply:
I thought about that as well. I mean it's not really a utility function. So I would either create a package `api` and put it in there or put it in `base`. I would rename it to something like `request_models.py`.